src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -72,16 +73,14 @@ public void add(T value, int index) {
                 addFront(value);
             }
         }
-        if (index > 0) {
-            if (index == size) {
-                addBack(value);
-            }
-            newNode = new Node(value);
-            nodeByIndex(index - 1).next = newNode;
-            newNode.next = nodeByIndex(index);
-            newNode.prev = nodeByIndex(index - 1);
-            nodeByIndex(index).prev = newNode;
+        if (index == size) {
+            addBack(value);
         }
+        Node<T> newNode = new Node(value);
+        nodeByIndex(index - 1).next = newNode;
```

Comment:
Node<T> node = nodeByIndex(index - 1);
node. next = newNode;
newNode.next = node.prev;
...

Reply:
done, but it was the same))